Apps/phterraformcloud/terraformcloud.json

```diff
@@ -0,0 +1,2914 @@
+                {
+                    "data_path": "action_result.data.*.data.*.attributes.status-timestamps.plan-queued-at",
+                    "data_type": "string",
+                    "example_values": [
+                        "2020-01-08T21:25:53+00:00"
+                    ]
+                },
+                {
+                    "data_path": "action_result.data.*.data.*.attributes.status-timestamps.plan-queueable-at",
+                    "data_type": "string",
+                    "example_values": [
+                        "2020-01-08T21:25:53+00:00"
+                    ]
+                },
+                {
+                    "data_path": "action_result.data.*.data.*.attributes.message",
+                    "data_type": "string",
+                    "example_values": [
+                        "Queued from Terraform Cloud UI"
+                    ],
+                    "column_name": "Message",
+                    "column_order": 5
+                },
+                {
+                    "data_path": "action_result.data.*.data.*.attributes.is-destroy",
+                    "data_type": "boolean",
+                    "example_values": [
+                        true,
+                        false
+                    ]
+                },
+                {
+                    "data_path": "action_result.data.*.data.*.attributes.permissions.can-force-execute",
+                    "data_type": "boolean",
+                    "example_values": [
+                        true,
+                        false
+                    ]
+                },
+                {
+                    "data_path": "action_result.data.*.data.*.attributes.permissions.can-force-cancel",
+                    "data_type": "boolean",
+                    "example_values": [
+                        true,
+                        false
+                    ]
+                },
+                {
+                    "data_path": "action_result.data.*.data.*.attributes.permissions.can-apply",
+                    "data_type": "boolean",
+                    "example_values": [
+                        true,
+                        false
+                    ]
+                },
+                {
+                    "data_path": "action_result.data.*.data.*.attributes.permissions.can-cancel",
+                    "data_type": "boolean",
+                    "example_values": [
+                        true,
+                        false
+                    ]
+                },
+                {
+                    "data_path": "action_result.data.*.data.*.attributes.permissions.can-discard",
+                    "data_type": "boolean",
+                    "example_values": [
+                        true,
+                        false
+                    ]
+                },
+                {
+                    "data_path": "action_result.data.*.data.*.attributes.permissions.can-override-policy-check",
```

Comment:
Can we remove one layer of nesting for `data.*.data.*` data paths?

Reply:
We can remove it as per other applications, but there are other keys like "meta" also included in the response. If we do so, we have to remove datapaths related to that. By looking at the datapaths, it looks like their response should be like the given below.
`{
     "data": [],
     "meta": {..},
     "links": {..}
}`
Hence, if we remove one layer, it will lead to the removal of meta and other info of the response. should we change it or keep it as it is? Please suggest.